hack/test-cmd.sh

```diff
@@ -690,6 +700,36 @@ runTests() {
   # Post-condition: no POD is running
   kube::test::get_object_assert 'pods --namespace=other' "{{range.items}}{{$id_field}}:{{end}}" ''
 
+  # Clean up 
+  kubectl delete namespace other
+
+  ##############
+  # Secrets #
+  ##############
+
+  ### Create a generic secret in a specific namespace
+  # Pre-condition: no SECRET exists
+  kube::test::get_object_assert 'secrets --namespace=test-secrets' "{{range.items}}{{$id_field}}:{{end}}" ''
```

Comment:
opened issue so its not lost: https://github.com/kubernetes/kubernetes/issues/18668